### ISSUE TITLE
Reset MSI

### ISSUE THREAD

**Issue by SamB-GB** [magento/inventory#2078]
Is there a way to fully reset MSI so that we can start again. We seem to be having an issue between the actual stock and the saleable quantity (which seems to double despite only having default stock enabled).

So I would like to reset MSI and start again with it.

**Comment by maghamed** [comment]
Hey @SamB-GB ,

Currently, there is no possibility of resetting MSI, because it depends on your particular business model how would you like to treat already opened reservations. Whether to wipe them out or not (as you will lose sensitive data). The same as we don't provide a possibility of resetting Orders in Magento. 

What I can recommend you to do is to check what you have in your reservation table.
```
mysql> select * from inventory_reservation;
+----------------+----------+--------------------+----------+---------------------------------------------------------------------+
| reservation_id | stock_id | sku                | quantity | metadata                                                            |
+----------------+----------+--------------------+----------+---------------------------------------------------------------------+
|              1 |        2 | testSimpleProduct2 | -15.0000 | {"event_type":"order_placed","object_type":"order","object_id":"1"} |
+----------------+----------+--------------------+----------+---------------------------------------------------------------------+
1 row in set (0.00 sec)
```
As data in Reservation table as well as your Min Quantity Threshold configuration could affect the resulting Salable Quantity. 

Take a few SKUs for which you have incorrect results and check the data for them in : 
-  inventory_source_item table `select * from inventory_source_item where sku = %sku%;`
- inventory_reservation table 
- check Min Qty Threshold 

In fact the calculation of salable quantity is quite a simple operation, you can see it here - https://github.com/magento-engcom/msi/blob/2.3-develop/app/code/Magento/InventorySales/Model/GetProductSalableQty.php#L75-L87

```
        $stockItemData = $this->getStockItemData->execute($sku, $stockId);
        if (null === $stockItemData || (bool)$stockItemData[GetStockItemDataInterface::IS_SALABLE] === false) {
            return 0;
        }

        $stockItemConfig = $this->getStockItemConfiguration->execute($sku, $stockId);
        $minQty = $stockItemConfig->getMinQty();

        $productQtyInStock = $stockItemData[GetStockItemDataInterface::QUANTITY]
            + $this->getReservationsQuantity->execute($sku, $stockId)
            - $minQty;
```

Did it help to solve your issue?

**Comment by maghamed** [comment]
should be addressed in #2140
